azurecaf/models.go

```diff
@@ -61,21 +63,36 @@ var Resources = map[string]ResourceStructure{
 	"app":    {"web app", "app", 2, 60, false, alphanumh, "^[0-9A-Za-z][0-9A-Za-z-]{0,58}[0-9a-zA-Z]$"},
 	"appi":   {"application insights", "appi", 1, 260, false, invappi, "^[^%&\\?/. ][^%&\\?/]{0,258}[^%&\\?/. ]$"},
 	"ase":    {"app service environment", "ase", 2, 37, false, alphanumh, "^[0-9A-Za-z-]{2,37}$"},
+	"asg":    {"application security group", "asg", 1, 80, false, alphanumhup, "^[0-9a-zA-Z][0-9A-Za-z_.-]{0,78}[0-9a-zA-Z_]$"},
```

Comment:
Thanks for the contribution! I've modified the structure in the latest release, could you add update merge the new changes and push?- I'll then integrate the changes

Reply:
Where would you like that i declare the new resource types in the model ? (models.go or in the json files)